imod/mf6/pkgbase.py

```diff
@@ -8,9 +9,42 @@
 
 import imod
 from imod.mf6.interfaces.ipackagebase import IPackageBase
+from imod.typing.grid import GridDataset
 
 TRANSPORT_PACKAGES = ("adv", "dsp", "ssm", "mst", "ist", "src")
 EXCHANGE_PACKAGES = "gwfgwf"
+ARGS_TO_EXCLUDE = ["validate"]
+
+
```

Comment:
would it make sense to relocate these functions out of packagebase and into a separate source file (for example of xugrid utilities)? Or into xugrid? They could be useful in other contexts.

Reply:
I now implemented it as a argument to the wrapper